GitUI/CommandsDialogs/FormRemotes.cs

```diff
@@ -247,10 +247,12 @@ private void InitialiseTabRemotes(string? preselectRemote = null)
                 Url.DataSource = repositoryHistory.ToList();
                 Url.DisplayMember = nameof(Repository.Path);
                 Url.SelectedItem = null;
+                Url.DropDownWidth = DpiUtil.Scale(600);
```

Comment:
Set to longest path instead?

Needed in Open repo form too...

Reply:
Done (and fixed the ones of the toolbar also).